docs/vmctl.md

```diff
@@ -815,12 +815,12 @@ Migration in `vm-native` mode takes two steps:
     --vm-native-src-addr=http://127.0.0.1:8481/select/0/prometheus \ # migrate from
     --vm-native-dst-addr=http://localhost:8428 \                     # migrate to
     --vm-native-filter-time-start='2022-11-20T00:00:00Z' \           # starting from
-    --vm-native-filter-match='{__name__=~"vm_cache_.*"}'             # only metrics matching the selector
+    --vm-native-filter-match='{__name__!~"vm_cache_.*"}'             # filter out metrics matching the selector
```

Comment:
I think it is better to change metric name to something neutral (to avoid inversion of matching expression) or change it to `!~"vm_.*"` to be aligned with recommendations.